### PR TITLE
Add ESLint Config Package

### PR DESCRIPTION
**Pull request by anthlasserre**

### CODE REVIEW

File: packages/eslint-config-react-native/package.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      "name": "Anthony Lasserre",
+      "url": "https://github.com/anthlasserre"
+    },
+    {
+      "name": "Mickael Dumand",
+      "url": "https://github.com/kimak"
+    }
+  ],
+  "bugs": {
+    "url": "https://github.com/GetLuko/eslint-plugin-react-native-guidelines/issues"
```
Comment (kimak) [inline packages/eslint-config-react-native/package.json:34]:
```suggestion
    "url": "https://github.com/GetLuko/eslint-react-native-guidelines/issues"
```

File: packages/eslint-config-react-native/package.json [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  },
+  "homepage": "https://github.com/GetLuko/eslint-plugin-react-native-guidelines",
```
Comment (kimak) [inline packages/eslint-config-react-native/package.json:36]:
```suggestion
  "homepage": "https://github.com/GetLuko/eslint-react-native-guidelines",
```

File: packages/eslint-config-react-native/src/index.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    './rules/getluko',
+    './rules/import',
+    './rules/prettier',
+    './rules/react-hooks',
+    './rules/react-native',
+    './rules/react',
+    './rules/typescript',
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+  ].map(require.resolve),
```
Comment (kimak) [inline packages/eslint-config-react-native/src/index.ts:16]: Where is coming this error?

File: packages/eslint-config-react-native/src/rules/import.ts [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+        ],
+        peerDependencies: true,
+        optionalDependencies: true,
+      },
+    ],
+    // Ensure imports point to a file/module that can be resolved
+    'import/no-unresolved': [
+      'error',
+      {
+        ignore: ['~/'], // @todo remove it to have better file unresolved detecion
```
Comment (kimak) [inline packages/eslint-config-react-native/src/rules/import.ts:48]: Should we create a ticket for this?

File: packages/eslint-config-react-native/src/rules/import.ts [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+        paths: [
+          {
+            name: 'react-native',
+            importNames: ['Dimensions'],
+            message:
+              "Use: import { deviceWidth, deviceHeight } from '~/luko-ui/theme/styles';",
+          },
+        ],
+      },
+    ],
```
Comment (kimak) [inline packages/eslint-config-react-native/src/rules/import.ts:71]: I would probably keep this rule in the project as it's coupled with the folder structure. WDYT?

**Review by kimak — CHANGES_REQUESTED** [review]

File: packages/eslint-config-react-native/src/rules/import.ts (hunk 4 above)
Comment (anthlasserre) [reply packages/eslint-config-react-native/src/rules/import.ts:48]: I don't know 🤷🏼‍♂️ I didn't really touch to the initial config. It was already there. Do you have in mind the context of this comment addition?

File: packages/eslint-config-react-native/src/rules/import.ts (hunk 5 above)
Comment (anthlasserre) [reply packages/eslint-config-react-native/src/rules/import.ts]: You're right finally. I moved the all import rules in our project 👍🏼

**Comment by hancandice** [comment]
We can just trick the `bundler` with `webpack.config.json`

[Point]

- Trick `ide` with `tsconfig.json` (bundler won't bundle if you cheat only `ide`)
- Try searching with this keyword: `tsconfig alias`

We can set with lines below 👇🏼 
```
tsconfig : alias
webpack.config: resolve.alias
```

[References] 

- https://www.typescriptlang.org/tsconfig#paths
- https://webpack.js.org/configuration/resolve/

**Pull request merged**